examples/tests/valid/dynamo_awscdk.w

```diff
@@ -0,0 +1,96 @@
+/*\
+skip: true
```

Comment:
@tsuf98 is this still going to run in our integration tests on aws?

Reply:
Only when running tests through hangar, we don't have it built-in in the CLI